lib.php

```diff
@@ -7293,7 +7293,7 @@ public function find_users($search) {
  * @param cm_info $cm Course-module object
  */
 function hsuforum_cm_info_view(cm_info $cm) {
-    if (!$cm->uservisible) {
+    if (!$cm->get_user_visible()) {
```

Comment:
@MatthewRorke - what is the difference between these two? Meaning the object attribute "uservisible", is that not always set hence the use for the function.

Reply:
According to the `cm_info` class. all of the attributes used, such as uservisible, are private attributes. As seen in `lib/modinfolib.php`